ruby24/plan.sh

```diff
@@ -0,0 +1,6 @@
+source ../ruby/plan.sh
```

Comment:
This is going to build this package as `ruby-2.4.3`,  rather than `ruby24-2.4.3`.  I think we want the latter, so that users can specify a dependency of `core/ruby24`.   Could you set the `pkg_name` here?

Reply:
Goooood catch. Missed that in my local build testing.

![I can fix it.](https://user-images.githubusercontent.com/517302/34623899-159c9660-f221-11e7-823e-07d3cdaf0da8.gif)